File: c++/pe006.cpp
// This is a solution for http://projecteuler.net/problem=6

#include <cstdio>

int main() {
  int sumsq = 0;
  for (int i = 0; i < 100; ++i) {
    sumsq += (i+1) * (i+1);
  }

  int sqsum = (101 * 50) * (101 * 50);
  printf("%d\n", sqsum - sumsq);

  return 0;
}

File: python/pe006.py
#!/usr/bin/env python3

""" this computes projecteuler.net problem 006 """

import sys

sumsq = 0
for i in range(1, 101):
    sumsq += i ** 2

SQSUM = (101 * 50) ** 2

print(SQSUM - sumsq)

sys.exit(0)
